src/i18n/en.js

```diff
@@ -153,6 +153,7 @@ You'll find an Email with steps to follow.`,
     download_msg: `Download`,
     error_msg: `There was an error gathering the invoices.`,
     no_data_msg: `There are no invoices to show`,
+    no_download_msg: `There is not invoice to download`,
```

Comment:
```suggestion
    no_download_msg: `There is no invoice to download`,
```

Reply:
Thanks Ceci, I will update the message in the next PR.